tests/kafkatest/tests/streams/streams_upgrade_test.py

```diff
@@ -53,7 +54,19 @@ def __init__(self, test_context):
             'wcnt' : { 'partitions': self.partitions, 'replication-factor': self.replication,
                        'configs': {"min.insync.replicas": self.isr} },
             'tagg' : { 'partitions': self.partitions, 'replication-factor': self.replication,
-                       'configs': {"min.insync.replicas": self.isr} }
+                       'configs': {"min.insync.replicas": self.isr} },
+            'SmokeTest-min-STATE-STORE-0000000008-changelog' : { 'partitions': self.partitions,
+                                                                 'replication-factor': self.replication,
+                                                                 'configs': {"min.insync.replicas": self.isr} },
+            'SmokeTest-max-STATE-STORE-0000000017-changelog' : { 'partitions': self.partitions,
+                                                                 'replication-factor': self.replication,
+                                                                 'configs': {"min.insync.replicas": self.isr} },
+            'SmokeTest-sum-STATE-STORE-0000000026-changelog' : { 'partitions': self.partitions,
+                                                                 'replication-factor': self.replication,
+                                                                 'configs': {"min.insync.replicas": self.isr} },
+            'SmokeTest-cnt-STATE-STORE-0000000035-changelog' : { 'partitions': self.partitions,
```

Comment:
Just playing devil's advocate here, do we want to create the internal topics ahead of time? As if we change the `SmokeTest` streams application at we'll need to update these again.

Reply:
We need to. We use the StreamsEosTestDriverService that expects a replication factor of 3, however, when we start up the 0.10.0.x version, it would by default create those topic with replication factor 1 and thus the test crashes later on.

Note, that we pull in the 0.10.0.1 code as-is and cannot update the Streams app there, to change the replication factor config to 3.